lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -472,6 +472,8 @@ def update(self, trans: GalaxyWebTransaction, id, payload, **kwds):
             new_workflow_name = workflow_dict.get("name")
             old_workflow = stored_workflow.latest_workflow
             name_updated = new_workflow_name and new_workflow_name != stored_workflow.name
+            if not name_updated:
```

Comment:
That doesn't look right, you should be able to e.g. only the hidden status.

Reply:
Ah my bad, yes this is in the wrong place entirely, moving it to the appropriate place(s)